src/main/java/addon/Storage.java

```diff
@@ -0,0 +1,84 @@
+package addon;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles storing and loading the save file.
+ */
+public class Storage {
+
+    /**
+     * Saves items in list in a local file
+     *
+     * @param list List to be written over.
+     */
+    public static void saveToFile(ArrayList<Task> list) {
+        File oldFile = new File("./save.txt");
+        boolean ignore = oldFile.delete();
+        try {
+            FileWriter fw = new FileWriter("./save.txt");
+            for (Task s : list) {
+                if (s instanceof Task.Todo) {
+                    fw.write("T" + "\r\n");
+                    fw.write(s.description + "\r\n");
+                } else if (s instanceof Task.Event) {
+                    Task.Event e = (Task.Event) s;
+                    fw.write("E" + "\r\n");
+                    fw.write(s.description + "\r\n");
+                    fw.write(e.date + "\r\n");
+                } else if (s instanceof Task.Deadline) {
+                    Task.Deadline e = (Task.Deadline) s;
+                    fw.write("D" + "\r\n");
+                    fw.write(s.description + "\r\n");
+                    fw.write(e.date + "\r\n");
+                }
+                fw.write((s.isDone ? "1" : "0") + "\r\n");
+            }
+            fw.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+
+    /**
+     * Loads the provided save file into the current Arraylist.
+     *
+     * @param saveFile Savefile to be loaded.
+     */
+    public static ArrayList<Task> loadFile(File saveFile) throws FileNotFoundException {
+
+        Scanner s = new Scanner(saveFile);
+        ArrayList<Task> list = new ArrayList<>();
+
+        while (s.hasNext()) {
+            String type = s.nextLine();
+            String name = s.nextLine();
```

Comment:
Perhaps naming these variables as taskType and taskName will make them more detailed and accurate?